java-agent/src/nativeMain/kotlin/com/epam/drill/agent/jvmti/event/VmInitEvent.kt

```diff
@@ -55,8 +55,8 @@ fun vmInitEvent(env: CPointer<jvmtiEnvVar>?, jniEnv: CPointer<JNIEnvVar>?, threa
     globalCallbacks()
     updatePackagePrefixesConfiguration()
     loadJvmModule("test2code")
-    WsSocket().connect(adminAddress.toString())
     RequestHolder.init(isAsync = agentParameters.isAsyncApp)
+    WsClient.connect(adminAddress.toString(), true)
```

Comment:
Will we ever pass `false`?

Reply:
This concerns connect() and reconnect() method, they use similar signature with async param with default false value for both methods. For connect() we are using async=true, but for reconnect we are leaving it default.

Reply:
Added for expressiveness - connect is always in the new thread, reconnect in "current" thread